packages/klighd-core/src/options/render-options-registry.ts

```diff
@@ -152,6 +208,8 @@ export class TextSimplificationThreshold implements RangeOption {
     }
     readonly stepSize = 0.1
     readonly initialValue: number = TextSimplificationThreshold.DEFAULT
+    readonly renderCategory: string = SmartZoom.ID
+    readonly description = 'The threshold font size to simplify text.\nIf set to 3 a text which is 3 or less pixel high is simplified.'
```

Comment:
For future reference: I think this option should be a debug option (possibility for that implemented in tik's bt code) with a reasonable value for illegible text sizes.

Reply:
I suggest making a ticket for this. Here is a bad spot for future references. #112